ssm_document/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "content_template" {}
```

Comment:
Maybe put a line break between the variables to make it more readable?

Reply:
I've put the line breaks in.